skabase/SKABaseDevice/SKABaseDevice.py

```diff
@@ -8,34 +8,37 @@
 """A generic base device for SKA. It exposes the generic attributes,
 properties and commands of an SKA device.
 """
+
+# PROTECTED REGION ID(SKABaseDevice.additionnal_import) ENABLED START #
+# standard imports
+import os
+import sys
+import logging
+import logging.handlers
+from logging.handlers import SysLogHandler
+import json
```

Comment:
Pep8 says line like `import foo` should be before lines like `from bar import baz`.

Reply:
Addressed